tests/ui/useless_attribute.stderr

```diff
@@ -7,17 +7,11 @@ LL | #[allow(dead_code)]
    = note: `-D clippy::useless-attribute` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(clippy::useless_attribute)]`
 
-error: useless lint attribute
-  --> $DIR/useless_attribute.rs:9:1
-   |
-LL | #[cfg_attr(feature = "cargo-clippy", allow(dead_code))]
-   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: if you just forgot a `!`, use: `#![cfg_attr(feature = "cargo-clippy", allow(dead_code)`
```

Comment:
This is an example fallout from removing the `feature=cargo-clippy` in the same release cycle.